report/renderer/report-ui-features.js

```diff
@@ -244,9 +244,11 @@ export class ReportUIFeatures {
 
     tablesWithUrls.forEach((tableEl) => {
       const rowEls = getTableRows(tableEl);
-      const primaryRowEls = rowEls.filter(rowEl => !rowEl.classList.contains('lh-sub-item-row'));
-      const thirdPartyRowEls = this._getThirdPartyRows(
-        primaryRowEls, Util.getFinalDisplayedUrl(this.json));
+      const nonSubItemRows = rowEls.filter(rowEl => !rowEl.classList.contains('lh-sub-item-row'));
+      const thirdPartyRowEls = this._getThirdPartyRows(nonSubItemRows,
+        Util.getFinalDisplayedUrl(this.json));
+      // Entity-grouped tables don't have zebra lines.
+      const hasZebraStyle = !rowEls.some(rowEl => rowEl.classList.contains('lh-row--group'));
```

Comment:
nit: just semantics at this point but could we change this to check for `lh-row--even`/`lh-row-odd` instead. The current logic assumes the only alternative to grouping is zebra styling which could theoretically change in the future.